pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -724,6 +732,13 @@ public static class Accounting {
     public static final String CONFIG_OF_PUBLISHING_JVM_USAGE = "accounting.publishing.jvm.heap.usage";
     public static final boolean DEFAULT_PUBLISHING_JVM_USAGE = false;
 
+    public static final String CONFIG_OF_CPU_TIME_BASED_KILLING_ENABLED = "accounting.cpu.time.based.killing.enabled";
+    public static final boolean DEFAULT_CPU_TIME_BASED_KILLING_ENABLED = false;
+
+    public static final String CONFIG_OF_CPU_TIME_BASED_KILLING_THRESHOLD_MS =
+        "accounting.cpu.time.based.killing.threshold.ms";
+    public static final int DEFAULT_CPU_TIME_BASED_KILLING_THRESHOLD_MS = 30_000;
```

Comment:
So the same configured value will apply to both broker and server ?

Reply:
Yes, it is instance level.